phpBB/includes/acp/acp_prune.php

```diff
@@ -177,7 +177,19 @@ function prune_forums($id, $mode)
 					'prune_old_polls'	=> $request->variable('prune_old_polls', 0),
 					'prune_announce'	=> $request->variable('prune_announce', 0),
 					'prune_sticky'		=> $request->variable('prune_sticky', 0),
-				)));
+				);
+
+				/**
+				 * Use this event to pass data from the prune form to the confirmation screen
+				 *
+				 * @event core.prune_forums_settings_confirm
+				 * @var int[]	hidden_fields	The IDs of the topics to be deleted
```

Comment:
I guess copy & paste error? This is an array and definitely does not just contain the topic IDs ;)